litmus-portal/backend/graphql-server/pkg/cluster/cluster_jwt.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/dgrijalva/jwt-go"
 )
 
-var secret = os.Getenv("CLUSTER_JWT_SECRET")
+var secret = os.Getenv("JWT_SECRET")
```

Comment:
It will create confusion with auth JWT

Reply:
It has to be the same as the auth server? else we won't be able to verify the user sessions in gql too.

Reply:
There will be two JWT
1. CLUSTER_JWT_SECET
2. AUTH_JWT_SECRET.

Reply:
The value of the 2 will have to be same, that's why I thought if keeping it same in both servers

Reply:
It's better to seperate these two JWT, as they both have different usecase

Reply:
and value will be also different

Reply:
The value cannot be different, because we will need to validate the tokens from the auth server in gql server too. To do that either we have to mention another jwt_secret env var or use the same one for this usecase too.